api/desecapi/views.py

```diff
@@ -198,19 +186,18 @@ def create(self, request, *args, **kwargs):
         if self.kwargs.get('subname') is not None or self.kwargs.get('type') is not None:
             raise MethodNotAllowed(method=request.method)
 
+        type = request.data.get('type', '').upper()
```

Comment:
regarding the permission, see https://github.com/desec-io/desec-stack/pull/50#discussion_r124585051